docker/cygnus-common/README.md

```diff
@@ -0,0 +1,164 @@
+#<a name="top"></a>cygnus-common docker
+Content:
+
+* [Before starting](#section1)
+* [Getting an image](#section2)
+    * [Building form sources](#section2.1)
+    * [Using docker hub image](#section2.2)
+* [Using the image](#section3)
+    * [As it is](#section3.1)
+    * [Using a specific configuration](#section3.2)
+        * [Editing the docker files](#section3.2.1)
+        * [Environment variables](#section3.2.2)
+        * [Using volumes](#section3.2.3)
+
+##<a name="section1"></a>Before starting
+Obviously, you will need docker installed and running in you machine. Please, check [this](https://docs.docker.com/linux/started/) official start guide.
+
+[Top](#top)
+
+##<a name="section2"></a>Getting an image
+###<a name="section2.1"></a>Building from sources
+Start by cloning the `fiware-cygnus` repository:
+
+    $ git clone https://github.com/telefonicaid/fiware-cygnus.git
+
+Change directory:
+
+    $ cd fiware-cygnus/docker/cygnus-common
+
+And run the following command:
+
+    $ docker build -t cygnus-common .
+
+Once finished (it may take a while), you can check the available images at your docker by typing:
+
+```
+$ docker images
+REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+cygnus-common       latest              0d2e537ac922        41 minutes ago      673.8 MB
+centos              6                   61bf77ab8841        6 weeks ago         228.9 MB                          
+```
+
+[Top](#top)
+
+###<a name="section2.2"></a>Using docker hub image
+Instead of building an image from the scratch, you may download it from [hub.docker.com](docker pull fiware/cygnus-common):
+
+    $ docker pull fiware/cygnus-common
+
+It can be listed the same way than above:
+
+```
+$ docker images
+REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+cygnus-common       latest              0d2e537ac922        41 minutes ago      673.8 MB
+centos              6                   61bf77ab8841        6 weeks ago         228.9 MB                          
+```
+
+[Top](#top)
+
+##<a name="section3"></a>Using the image
+###<a name="section3.1"></a>As it is
+The cygnus-common image (either built from the scratch, either downloaded from hub.docker.com) allows running a Cygnus agent in charge of log messages at INFO level, known as [logger-sink](https://flume.apache.org/FlumeUserGuide.html#logger-sink).
```

Comment:
know as --> ". This is because the default agent configuration runs a logger sink".